src/utils/datetime.js

```diff
@@ -68,3 +68,17 @@ export function formatDurationToClocktime(duration) {
     timeOfDay: hoursPlus[1],
   };
 }
+
+export function formatDisplayDate(timestamp) {
+  if (!timestamp) {
+    return '';
+  }
+  return moment(timestamp).utc().format('MMM Do YYYY');
+}
+
+export function millisecondsAsTimeOfDay(milliseconds) {
+  if (milliseconds === null || milliseconds === undefined) {
+    return '';
+  }
+  return moment(milliseconds).utc().format('hh:mm a');
```

Comment:
You have duplicated here. My `formatDurationToClocktime` serves the same purpose, although my use case was a bit more complex so it provides the result a little differently. Let's discuss how to consolidate for all use cases.